src/actions.ts

```diff
@@ -143,13 +143,25 @@ export function toActivityDefinition<TContext, TEvent extends EventObject>(
  */
 export function raise<TContext, TEvent extends EventObject>(
   event: Event<TEvent>
-): RaiseEvent<TContext, TEvent> {
+): RaiseAction<TEvent> | SendAction<TContext, TEvent> {
+  if (!isString(event)) {
```

Comment:
wasn't 100% sure how to approach this, weirdly SCXML allows only "simple" events in raise, so I've kept that - but at the same time we can create more complex events to be raised using `<send/>` with internal target

so I've decided to keep SCXML semantics so conversion can be accurate, but also made our `raise` "smarter" so it can handle complex events, just by using send internally instead of raise